src/lib/nifindexes.nim

```diff
@@ -249,7 +249,8 @@ proc createIndex*(infile: string; root: PackedLineInfo; buildChecksum: bool; sec
     content.add "(checksum \"" & $final & "\")"
   content.add "\n)\n"
   if fileExists(indexName) and readFile(indexName) == content:
-    discard "no change"
+    discard "Only change the last modification time so that make command works correctly"
```

Comment:
Instead just remove the entire `if fileExists(indexName) and readFile(indexName) == content` and always write the file.

Reply:
Done